server/src/services/RedisClient.js

```diff
@@ -30,13 +29,37 @@ class RedisClient {
         return await recursiveScan();
     }
 
-    jsonGet(key) {
+    async jsonGet(key) {
         return this.redis.json_get(key);
     }
 
-    jsonSet(key, path, json) {
+    async jsonSet(key, path, json) {
         return this.redis.json_set(key, path, json);
     }
+
+    async hgetall(key) {
```

Comment:
remove async if there is no await inside

Reply:
done